portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,79 @@
+@import url('https://fonts.googleapis.com/css2?family=Cabin&family=Raleway:wght@500&display=swap');
+body {
+    /* background-color: rgb(102, 194, 143); */
+    /* font-family: 'Raleway', sans-serif; */
```

Comment:
It is usually better to delete commented code but if you think you will use it in the near future feel free to keep it in.

Reply:
Deleted!